openedx/core/djangoapps/content/search/documents.py

```diff
@@ -225,6 +229,15 @@ class implementation returns only:
             }
             if cur_block.scope_ids.block_type != "course":
                 parent_data["usage_key"] = str(cur_block.usage_key)
+                # Section and subsections don't have URLs in the CMS
+                if cur_block.scope_ids.block_type not in ["chapter", "sequential"]:
+                    parent_data["url"] = (
+                        reverse_course_url("course_handler", cur_block.context_key) +
+                        reverse_usage_url("container_handler", cur_block.usage_key)
+                    )
+            else:
+                parent_data["url"] = reverse_course_url("course_handler", cur_block.context_key)
```

Comment:
Not sure if we need to include this data in index as it can be easily derived from the information already present.

Reply:
You're right. Initially, I thought this would help us handle different waffle flags (like `contentstore.new_studio_mfe.use_new_unit_page`) but that is not the case.

Reverted here: 6b27c090346a28ad6d427e172f703f10616b7fb8